client/src/interface/User.ts

```diff
@@ -63,5 +64,10 @@ export interface SearchUsersApiData {
 }
 
 export interface Customer {
```

Comment:
Also, are we using this interface at the moment?

Reply:
We are not, removing for now